src/azure-cli/azure/cli/command_modules/databoxedge/manual/_help.py

```diff
@@ -0,0 +1,326 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+
+helps['databoxedge'] = """
+    type: group
+    short-summary: Support data box edge device and management
+"""
+
+helps['databoxedge device'] = """
+    type: group
+    short-summary: Manage device with databoxedge
+"""
+
+helps['databoxedge device list'] = """
+    type: command
+    short-summary: "Get all the Data Box Edge/Data Box Gateway devices in a resource group or in a subscription."
+    examples:
+      - name: Get all the Data Box Edge/Data Box Gateway devices in a resource group.
+        text: |-
+               az databoxedge device list --resource-group "GroupForEdgeAutomation"
+      - name: Get all the Data Box Edge/Data Box Gateway devices in a subscription.
+        text: |-
+               az databoxedge device list
+"""
+
+helps['databoxedge device show'] = """
+    type: command
+    short-summary: "Get the properties of the Data Box Edge/Data Box Gateway device."
+    examples:
+      - name: Get the properties of the Data Box Edge/Data Box Gateway device.
+        text: |-
+               az databoxedge device show --name "testedgedevice" --resource-group "GroupForEdgeAutomation"
+"""
+
+helps['databoxedge device create'] = """
+    type: command
+    short-summary: "Create a Data Box Edge/Data Box Gateway resource."
+    parameters:
+      - name: --sku
+        short-summary: "The SKU type."
+        long-summary: |
+            Usage: --sku name=XX tier=XX
+
+            name: SKU name.
+            tier: The SKU tier. This is based on the SKU name.
```

Comment:
For sku issue, it seems the tier is based on name. So I am thinking if we could get sku type with name only.
With current design, it is a little complicated for customer to define their sku and they have no idea for allowed value.

Reply:
modified